addons/sale/static/src/scss/sale_report.scss

```diff
@@ -1,3 +1,9 @@
 .sale_tbody .o_line_note {
     word-break: break-word;
 }
+
+/*rtl:raw:
+.offset-1 {
+    margin-left: 0% !important;
+}
+*/
```

Comment:
remove if no longer necessary?

Reply:
It is necessary to solve the issue without updating the module. The plan is to remove this part of the fix from `master`.